{
    "error": {
        "id": 31,
        "message": "Username already in use"
    }
}